CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### v3.4.0 (2023-07-11)
+
+- Make bindReporter generic over metric type ([#359](https://github.com/GoogleChrome/web-vitals/pull/359))
+- Update INP status in README ([#362](https://github.com/GoogleChrome/web-vitals/pull/362))
+- Metric type fixes ([#356](https://github.com/GoogleChrome/web-vitals/pull/356))
```

Comment:
Nit: can this be reworded to be consistent with the other entries (e.g. where the first word is an active verb)?

Reply:
Done.